pkg/subctl/cmd/join.go

```diff
@@ -38,7 +38,7 @@ var (
 	serviceCIDR     string
 	clusterCIDR     string
 	repository      string
-	version         string
+	imageVersion    string
```

Comment:
This field is modified to avoid overlap with the pkg import happening on version.go over the same package:

"github.com/submariner-io/submariner-operator/pkg/version"